integration-cli/docker_api_exec_test.go

```diff
@@ -108,6 +99,30 @@ func (s *DockerSuite) TestExecAPIStartBackwardsCompatible(c *check.C) {
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusOK, comment)
 }
 
+// #19362
```

Comment:
perhaps this should have a short description and the full URL to the issue (in case code is moved around or read outside of github)